GetMerchant/handler.ts

```diff
@@ -149,8 +149,11 @@ export const GetMerchantHandler = (
                 O.fold(
                   () =>
                     pipe(
-                      d.landing_page_referrer,
+                      d.landing_page_url,
                       O.fromNullable,
+                      O.chain(__ =>
+                        pipe(d.landing_page_referrer, O.fromNullable)
```

Comment:
From the PR description, shouldn't it be like the following code to have a default `-` on missing `landing_page_referrer`?
```suggestion
                        pipe(
                          c.url,
                          O.fromNullable,
                          O.fold(
                            () => O.some("-"),
                            O.some
                          )
                        )
                      ),
```

Reply:
My bad. Removed after a refactor. Fixed in https://github.com/pagopa/io-functions-cgn-operator-search/pull/65/commits/03e3b4a48475ee86122b383981557787afa13334